addons/account/wizard/account_payment_register.py

```diff
@@ -149,15 +148,53 @@ def _get_batches(self):
             raise UserError(_("You can't open the register payment wizard without at least one receivable/payable line."))
 
         batches = {}
-        for line in lines:
-            batch_key = self._get_line_batch_key(line)
-
-            serialized_key = '-'.join(str(v) for v in batch_key.values())
+        flagged_keys = ()
```

Comment:
`flagged_keys` isn't really self-explaining as variable name. Would be better to know why it's flagged and in there when looking that name